build/Common.prod.props

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)/OpenTelemetry.prod.ruleset</CodeAnalysisRuleSet>
     <ExposeExperimentalFeatures Condition="'$(ExposeExperimentalFeatures)' == ''">true</ExposeExperimentalFeatures>
+    <EnablePackageValidation Condition="'$(EnablePackageValidation)' == ''">false</EnablePackageValidation>
```

Comment:
nit: package validation is an opt-in feature. No need to specify it here. Or am I missing something?

```suggestion
```

Reply:
This is just a convenience thing so that when someone wants to test package validation from IDE we can toggle it to true and then kick off a package.

Reply:
I see. Is there any reason why you don't want to enable it by default? That's what we usually recommend.

Reply:
I don't feel passionately it shouldn't be ON by default. My thinking was, we have the public api analyzer currently. That's what we all suffer through at dev-time 😄 The CI also runs a pass through the package validation. If we turn this on by default, I don't think it will add much benefit. Because I don't think most devs run a pack ever. I think package validation only fires during pack? Basically it will just add some cycles at restore to fetch the stable versions which didn't seem crucial or really beneficial so I left it OFF.

Reply:
Fetching a baseline package during restore should be super fast and not be noticeable (especially as it's cached after the first restore) but I understand where you're coming from. Thanks for the clarification :)